Cleanse/BinderArities.swift

```diff
@@ -1,12 +1,12 @@
 /// DO NOT EDIT - This file was auto-generated by CleanseGen/main.swift
 extension BindToable {
 
-/// 1-arity `to1(factory:)` function.
+/// 1-arity `to(factory:)` function.
 /// This completes the binding process for registering the provider for the type passed (or inferred) to `bind()`.
 /// - parameter factory: Takes arguments required to construct `Element` passed to the `bind()` function
 ///
 /// - Note: This method was auto-generated by CleanseGen/main.swift
-@discardableResult public func to1<P_1>(file: StaticString=#file, line: Int=#line, function: StaticString=#function, factory: @escaping (P_1) -> Input)  -> BindingReceipt<Input> {
```

Comment:
@holmes Look at the diff here. In particular just adding an extra set of parens fixing it so `factory: @escaping ((P_1)) -> Input)` instead of the old `factory: @escaping (P_1) -> Input)`

Reply:
That's amazing